src/components/Input/Textarea.test.tsx

```diff
@@ -0,0 +1,63 @@
+
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import { Textarea } from './Textarea';
+
+describe('<Texarea />', () => {
+    describe("Rendering", () => {
+        it('should render without throwing an error', () => {
+            const cut = shallow(<Textarea label="Label" />);
+            expect(cut.find('textarea')).toHaveLength(1);
+        });
+        it('should render the label', () => {
+            const cut = shallow(<Textarea label="Label" />);
+            expect(cut.find('label').text()).toEqual("Label");
+        });
+        it('should associate the label with the input', () => {
+            const cut = shallow(<Textarea id="the_id" label="Label" />);
+            expect(cut.find('textarea').prop('id')).toEqual("the_id");
+            expect(cut.find('label').prop('htmlFor')).toEqual("the_id");
+        });
+        it('should apply custom class', () => {
+            const cut = shallow(<Textarea className="my-class" label="Label" />);
+            expect(cut.find('div.rvt-input').hasClass("my-class")).toEqual(true);
+        });
+    });
+    describe("Notes", () => {
+        it('should apply the note', () => {
+            const cut = shallow(<Textarea note="Note" label="Label" />);
```

Comment:
note attribute to me is not entirely obvious.. at first I thought it might be the default value of the textarea (is default value an attribute?) but then realized it is just a note about the text input. This is interesting and makes me wonder if we should add validation pieces to the components.. 1.) the rule for validation and 2.) the alert type and message for invalid fields..